DeleteCFStack/index.js

```diff
@@ -1,64 +1,75 @@
 const AWS = require('aws-sdk');
 const cf = new AWS.CloudFormation();
 
-async function handleWebhookEvent(webhookBody) {
-    const action = webhookBody['action'];
-    const pullRequest = webhookBody['pull_request'];
-    let message = 'Event ignored';
+function parsewebhookBody(eventType, webhookBody) {
```

Comment:
Small stylistic notes - this could be `parseWebhookBody` (camelcase) here and in DeleteS3FeatureBucket, and on line 10 we should use `!== master` (double ==)